f8a_worker/utils.py

```diff
@@ -596,3 +598,28 @@ def peek(iterable):
     except StopIteration:
         return None
     return first
+
+
+@retry(reraise=True, stop=tenacity.stop_after_attempt(3), wait=tenacity.wait_fixed(1))
+def get_gh_contributors(url):
+    """Get number of contributors from Git URL.
+
+    :param url: URL where to do the request
+    :return:  length of contributor's list
+    """
+    try:
+        url += "?per_page=1"
+        response = requests.head(url)
+        response.raise_for_status()
+
+        if response.status_code == 204:
+            raise HTTPError('No content')
+        elif response.status_code == 200:
+            contributors_count = int(parse_qs(response.links['last']['url'])['page'][0])
+            return contributors_count
+        else:
+            return 0
+    except HTTPError as err:
+        message = "Failed to get results from {url} with {err}".format(url=url, err=err)
```

Comment:
I guess the url would be already part of the exception message(HTTPError). You can remove it.